Monika After Story/game/script-topics.rpy

```diff
@@ -7591,6 +7591,127 @@ label monika_cartravel:
+
+        "Yuri":
+            m 1hua "All right."
+            m 1eua "And which poem?"
+            menu:
+                m "And which poem?{fast}"
+                "{i}Ghost under the light, pt.1{/i}":
+                    m 1duc "Hold on..."
+                    call showpoem(poem=poem_y1,music=False) from _call_showpoem_7 #yuri1
+                
+                "{i}Ghost under the light, pt.2{/i}":
+                    m 1duc "Hold on..."
+                    call showpoem(poem=poem_y3b,music=False) from _call_showpoem_7 #yuri1bis
+                    
+                "{i}The racoon{/i}":
+                    m 1duc "Hold on..."
+                    call showpoem(poem=poem_y2,music=False) from _call_showpoem_7 #yuri2
+                    
+                "{i}Beach{/i}":
+                    m 1duc "Hold on..."
+                    call showpoem(poem=poem_y3,music=False) from _call_showpoem_7 #yuri3
+        "Sayori":
+            m 1hua "Oh, all right."
+            m 1eua "And which poem?"
+            menu:
+                m "And which poem?{fast}"
+                "{i}Dear sunshine{/i}":
+                    m 1duc "Hold on..."
+                    call showpoem(poem=poem_s1,music=False) from _call_showpoem_7 #sayo1
+                
+                "{i}Bottles{/i}":
+                    m 1duc "Hold on..."
+                    call showpoem(poem=poem_s2,music=False) from _call_showpoem_7 #sayo2
+                    
+                "{i}Get out{/i}":
+                    m 1ekc "Oh...{w} Well, it's as you want."
+                    m 1dkc "Hold on..."
+                    call showpoem(poem=poem_s3,music=False) from _call_showpoem_7 #sayo3
+                    m 1ekc "I really don't like this one."
+                    m 1rkc "I mean, it was the last poem before her..."
+                    m 1dkc "You know what I mean."
+                    m 1eua "But whatever."
+                    
+
+        "Natsuki":
+            m 1hksdlb "Ahah, cuteness is back."
+            m 1eua "And which poem?"
+            menu:
+                m "And which poem?{fast}"
+                "{i}Eagle can fly{/i}":
+                    m 1duc "Hold on..."
+                    call showpoem(poem=poem_n1,music=False) from _call_showpoem_7 #nat1
+                
+                "{i}Amy like spiders{/i}":
+                    m 1duc "Hold on..."
+                    call showpoem(poem=poem_n2,music=False) from _call_showpoem_7 #nat2
+                    
+                "{i}I'll be your beach{/i}":
+                    m 1duc "Hold on..."
+                    call showpoem(poem=poem_n3,music=False) from _call_showpoem_7 #nat3
+                    
+                "{i}Because you{/i}":
+                    m 1duc "Hold on..."
+                    call showpoem(poem=poem_n3b,music=False) from _call_showpoem_7 #nat3b
+                    
+        "Monika":
+            m 1hub "Yeah, you picked me!"
+            m 1ttu "I know, my poems are the best..."
```

Comment:
Did you mean to put a comma there? It's fine both ways, but it makes a difference in how it's said.

Reply:
Yeah, it's not a mistake : I thought that it suit Monika's voice better than without it